app/src/main/java/de/tum/in/tumcampusapp/database/Converters.kt

```diff
@@ -0,0 +1,29 @@
+package de.tum.`in`.tumcampusapp.database
+
+import android.arch.persistence.room.TypeConverter
+import com.google.gson.Gson
+import de.tum.`in`.tumcampusapp.auxiliary.Utils
+import de.tum.`in`.tumcampusapp.models.gcm.GCMNotificationLocation
+import java.util.*
+
+class Converters {
+    @TypeConverter
+    fun fromIso(str: String): Date {
+        return Utils.getDateTime(str)
```

Comment:
please use `fun fromIso(str) = Utils.getDateTime(str)` for [single expression functions](https://www.packtpub.com/mapt/book/application_development/9781787126367/4/ch04lvl1sec49/single-expression-functions)